### ISSUE TITLE
[stable/spark] Unable to launch Zeppelin deployment locally

### ISSUE THREAD

**Issue by TeemuKoivisto** [helm/charts#9558]
**Is this a request for help?**:
I guess?
---

**Is this a BUG REPORT or FEATURE REQUEST?** (choose one):
BUG REPORT

**Version of Helm and Kubernetes**:
2.10.0
1.10.0 (using local minikube 0.25.0)

**Which chart**:
stable/spark

**What happened**:
Unable to launch Zeppelin deployment/pod.

**What you expected to happen**:
I would presume the notebook should start. If this is a configuration issue it's very unclear **what** I should do to fix it.

**How to reproduce it** (as minimally and precisely as possible):
For me it's simply by installing and running the chart: `helm install stable/spark`

**Anything else we need to know**:
This is the error message it gives me:
`
Unable to mount volumes for pod "intent-squid-zeppelin-6ffb7d6bb6-crsnd_default(b121e854-f0ff-11e8-9931-080027339206)": timeout expired waiting for volumes to attach/mount for pod "default"/"intent-squid-zeppelin-6ffb7d6bb6-crsnd". list of unattached/unmounted volumes=[intent-squid-master-config intent-squid-master-notebook]
`

EDIT: Now that I upgraded to minikube 0.30.0 and in the process deleted my old minikube installation, the error the Zeppelin deployment now gives is: `persistentvolumeclaim "cranky-fish-zeppelin-config" not found`

**Comment by TeemuKoivisto** [comment]
So the problem was not explicitly setting Persistence to false for both volumes. Which is strange since the false is the default value of the chart. I'd wager that there is something wrong with the chart unless I am terribly wrong.

```bash
helm install \
  --set Zeppelin.Persistence.Config.Enabled=false \
  --set Zeppelin.Persistence.Notebook.Enabled=false \
  --set Zeppelin.ServiceType=NodePort \
  --set Master.ServiceType=NodePort \
  stable/spark
```

That one worked although in the end I ended up using local volume that I switched manually in the YAML of the Zeppelin deployment. I'll be looking into the chart to see what causes the error unless somebody who has more time and experience about this matter can help me out.
